ps_crossselling.php

```diff
@@ -246,7 +246,8 @@ protected function getOrderProducts(array $productIds = [])
         FROM ' . _DB_PREFIX_ . 'orders o
         LEFT JOIN ' . _DB_PREFIX_ . 'order_detail od ON (od.id_order = o.id_order)
         WHERE o.valid = 1
-        AND od.product_id IN (' . implode(',', $productIds) . ')';
+        AND od.product_id IN (' . implode(',', $productIds) . ')
+        ORDER BY date_add desc LIMIT ' . ((int) Configuration::get('CROSSSELLING_NBR')) * 50;
```

Comment:
Would be a good idea to use alias to avoid any SQL conflict in the future:
```suggestion
        ORDER BY o.date_add desc LIMIT ' . ((int) Configuration::get('CROSSSELLING_NBR')) * 50;
```